src/main/java/uk/ac/ebi/subs/fileupload/errors/SubmissionNotExistsException.java

```diff
@@ -0,0 +1,10 @@
+package uk.ac.ebi.subs.fileupload.errors;
+
+public class SubmissionNotExistsException extends RuntimeException {
```

Comment:
Wouldn't `SubmissionNotFoundException` be a more appropriate exception name?

Reply:
Agree